docs/userguide/introduction.rst

```diff
@@ -1,15 +1,14 @@
 Introduction
 ============
 
-Welcome to ZoneMinder, the all-in-one Linux GPL'd security camera solution.
+Welcome to ZoneMinder, the all-in-one security camera solution for Linux with GPL License.
 
-Most commercial "security systems" are designed as a monitoring system that also records.  Recording quality can vary from bad to unusable, locating the relevant video can range from challenging to impractical, and exporting can often only be done with the manual present.  ZoneMinder was designed primarily to record, and allow easy searches and exporting.  Recordings are of the best possible quality, easy to filter and find, and simple to export using any system with a web browser.  It also monitors.
+Commercial "security systems" are usually designed as a monitoring system with unusable recording quality. In such a system, locating and exporting relevant video can be challenging and often requires human intervention.  ZoneMinder provides the best possible record quality and allows easy searches, filtering and exporting of security footage. 
```

Comment:
I feel that this first statement goes a little too far.  How about often instead of usually.

Reply:
Thank you for your feedback! I think the first statement is a quite bold claim. I will change "usually" with often.